compose/cli/main.py

```diff
@@ -318,15 +318,15 @@ def config(self, options):
         """
         Validate and view the Compose file.
 
-        Usage: config [options]
+        Usage: config [options] [SERVICE...]
```

Comment:
This is going to be really awkward, because `docker-compose config service1` will be a no-op where people could reasonably expect an output containing only the specified service(s) and dependent resources.
I'm not suggesting implementing that (it would be a significant amount of work for a mostly useless option), but I'd like to have a better way to make sure it's obvious the filtering only applies to `--hash`.

1. Make `--hash` a string flag that accepts a comma-separated list of services (leave blank for all services)
2. Remove the option and just let users grep for the services they want to process.

I'm slightly partial to 2 because it's more UNIXy, but 1 may be reasonable too.